src/vip_ipykernel_launcher.py

```diff
@@ -0,0 +1,26 @@
+"""Taken and modified from
+https://github.com/ipython/ipykernel/blob/master/ipykernel_launcher.py
+
+Entry point for launching an ViP-IPython kernel.
+"""
+
+import subprocess
+import sys
+
+from vip_ipykernel import venv_search
+
+if __name__ == '__main__':
+    # Remove the CWD from sys.path while we load stuff.
+    # This is added back by InteractiveShellApp.init_path()
+    if sys.path[0] == '':
+        del sys.path[0]
```

Comment:
N.B. `ipykernel_launcher` imports ipykernel after this sys.path check - that's the point of having a separate `ipykernel_launcher` module. It avoids ipykernel failing to even launch because someone has made `random.py` or something in the CWD.

Reply:
Aha, yes I didn't fully understand why this was there to be honest. Given that this, in the end, just calls `ipykernel_launcher`, I take it that it would be fine to remove the sys.path check?

Reply:
It's less of an issue than with ipykernel, because you import so much less stuff, but anything you do import (like pathlib) could be shadowed by files in the cwd. So the `_launcher` workaround  removes that from `sys.path` before any imports, and then the ipykernel machinery adds it back later (once most of its own imports are done).

Reply:
Ooooh I get it now, huh yeah that's an interesting issue, I'll leave it as it is for now then, thank you for the explanation

Reply:
If you're leaving this module around, you probably want to move the `from vip_ipykernel` import after this line. Otherwise this code isn't really doing anything.